src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/admin/staff/CourtWorkerController.java

```diff
@@ -0,0 +1,125 @@
+
+@Slf4j
+@RestController
+@RequestMapping("/admin/staff")
+@RequiredArgsConstructor
+public class CourtWorkerController {
+
+    private final VerifyTokenService verifyTokenService;
+    private final CourtWorkerService courtWorkerService;
+
+    @PostMapping(value = "/addCourtWorker", consumes = APPLICATION_JSON_VALUE)
+    @Operation(summary = "Add a court worker")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "Accessed successfully"),
+        @ApiResponse(responseCode = "400", description = "Bad Request"),
+        @ApiResponse(responseCode = "500", description = "Internal Server Error")
+    })
+    public ResponseEntity<CCDCallbackResponse> addCourtWorker(
+            @RequestHeader(HttpHeaders.AUTHORIZATION) String userToken,
+            @RequestBody CCDRequest ccdRequest) {
+
+        log.info("/addCourtWorker");
+
+        if (!verifyTokenService.verifyTokenSignature(userToken)) {
+            return ResponseEntity.status(HttpStatus.FORBIDDEN.value()).build();
+        }
+
+        var adminData = ccdRequest.getCaseDetails().getAdminData();
+        List<String> errors = courtWorkerService.addCourtWorker(adminData);
+
+        return CCDCallbackResponse.getCallbackRespEntityErrors(errors, adminData);
+    }
+
+    @PostMapping(value = "/updateCourtWorkerMidEventOffice", consumes = APPLICATION_JSON_VALUE)
+    @Operation(summary = "Populates the dynamicList for court worker when office and type selected")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "Accessed successfully"),
+        @ApiResponse(responseCode = "400", description = "Bad Request"),
+        @ApiResponse(responseCode = "500", description = "Internal Server Error")
+    })
+    public ResponseEntity<CCDCallbackResponse> updateCourtWorkerMidEventOffice(
+            @RequestHeader("Authorization") String userToken,
+            @RequestBody CCDRequest ccdRequest) {
+
+        log.info("/updateCourtWorkerMidEventOffice");
+
+        if (!verifyTokenService.verifyTokenSignature(userToken)) {
+            return ResponseEntity.status(HttpStatus.FORBIDDEN.value()).build();
+        }
+
+        var adminData = ccdRequest.getCaseDetails().getAdminData();
+        List<String> errors = courtWorkerService.updateCourtWorkerMidEventOffice(adminData);
+
+        return CCDCallbackResponse.getCallbackRespEntityErrors(errors, adminData);
+    }
+
+    @PostMapping(value = "/updateCourtWorkerMidEventClerk", consumes = APPLICATION_JSON_VALUE)
+    @Operation(summary = "Populates the dynamicList for court worker when office and type selected")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "Accessed successfully"),
+        @ApiResponse(responseCode = "400", description = "Bad Request"),
+        @ApiResponse(responseCode = "500", description = "Internal Server Error")
+    })
+    public ResponseEntity<CCDCallbackResponse> updateCourtWorkerMidEventClerk(
```

Comment:
clerk to courtworker, and in the postMapping

Reply:
Thank you Harpreet